config/manifests/bases/kustomization.yaml

```diff
@@ -1,6 +1,11 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - cephcsi-operator.clusterserviceversion.yaml
+- cephcsi-operator.clusterserviceversion.yaml
 commonAnnotations:
   olm.skipRange: ""
+patches:
```

Comment:
@Nikhil-Ladha we already have these changes in main(we recently done changes to the downstream main branch only). are we good to close this PR and also do we need to sync sync-up-main branch? because this are only downstream changes